.idea/encodings.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
is it possible to leave this one out and rely on the editorconfig charset setting instead? 

To avoid having to update this on each module addition/ deletion